### ISSUE TITLE
Activity item drawer not translated

### ISSUE THREAD

**Issue by licitdev** [directus/directus#8415]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

Activity item drawer isn't translated.
<img width="821" alt="Screenshot 2021-09-29 at 5 26 01 PM" src="https://user-images.githubusercontent.com/26413686/135243361-13cdce3b-5eb0-4004-acd2-e18f3baf6178.png">

### To Reproduce

1. Change user's language to anything other than English
2. Open Activity Feed
3. Clicking on any entry
4. See Activity Item drawer remains to be in English

### What version of Directus are you using?

v9.0.0-rc.95

### What version of Node.js are you using?

v16

### What database are you using?

Postgres

### What browser are you using?

Safari

### What operating system are you using?

macOS

### How are you deploying Directus?

locally

**Comment by benhaynes** [comment]
I'd like to either:

A. Clean this drawer up with a dedicated activity detail design
B. Not have a drawer, and instead, find a place for this data on the Browse Activity page. See the design below that @Nitwel is working on. This shows most info already... and can show ALL info if we use the INFO icon tooltip for IP Address and User-Agent. This approach works because activity items can't be modified anyway, so no need for a drawer/form.

Thoughts on which is better?

![Screen Shot 2021-09-29 at 8 51 06 AM](https://user-images.githubusercontent.com/522079/135271883-7d068db8-1ace-4219-b58a-b8474739eed5.png)

**Comment by rijkvanzanten** [comment]
There's a bit more info in that drawer currently than we're showing on the row. If we can find a way to show those data points in a tooltip/popover or some other place on the row, I'm in favor of getting rid of the drawer completely 👍🏻

**Comment by azrikahar** [comment]
Definitely in favor of removing the need of drawer for me as well! Likely "less friction" compared to drawers. Btw this "favoring tooltip over drawer" sounds familiar to me, vaguely recall you guys discussed about this somewhere, but can't remember where..

With that said, should we resolve this issue first since it seems like the translations are already there? 

https://github.com/directus/directus/blob/cb7b6a91084d0c71e886a8d0481280ad066a6e63/app/src/lang/translations/en-US.yaml#L838-L847

**Comment by benhaynes** [comment]
Yes, let's resolve this one now since that's easy. I'll try to find some time with Rijk to wrap up the new layout.

**Comment by erondpowell** [comment]
The idea of having a little metrics panel is really nice. It would be cool if that was filterable as well... displaying logins, create, edits, deletes, or specific user activity. Just brainstorming!
